lib/widget/charts/line.js

```diff
@@ -115,6 +114,21 @@ Line.prototype.setData = function(data) {
     return formatYLabel(max, max, min, numLabels, wholeNumbersOnly, abbreviate).length * 2;
   }
 
+  function getMax(collection, iteratee) {
```

Comment:
for the existing use-cases we have I didn't see it important to implement any robust error checking or handling